### ISSUE TITLE
cannot leave a room

### ISSUE THREAD

**Issue by wollow22** [vector-im/riot-web#7744]
### can't leave riot android

Recently I visited the room: Riot Android. I left it after some time, but now it keeps haunting me. I can't delete it from my room list on the desktop PC. I am receiving a message that there is a message for me. When I open Riot android then I can see that the last message is 14 days old and says that I left the room. If I join other rooms I can leave them after leaving they get moved to Historical and there I can choose to forget them...

### Steps to reproduce

- For bugs, list the steps
- that reproduce the bug
- using hyphens as bullet points

Describe how what happens differs from what you expected.

Log: sent/not sent?    

### Version information

- **Platform**: Palemoon 28desktop?

For the web app:

- **Browser**: Chrome, Safari, Firefox? which version?
- **OS**: Windows, macOS, Ubuntu, Arch Linux, etc?
- **URL**: riot.im/develop / riot.im/app / somewhere else? If a private server, what version of riot-web?

For the desktop app:
web (in-browser) or
- **mx17 64**
- **Version**: 0.x.y

**Comment by wollow22** [comment]
![frozen riot android](https://user-images.githubusercontent.com/25129872/48975945-2af2bd00-f0b8-11e8-8fca-55309fb86f82.png)
Sorry for the incomplete issue report. It was the first time and it seems I need practice.
Meanwhile I was able to leave the room. The room issues was on the desktop. Later I realized that the room was not open on the android. I logged in via android with the riot app. Joined the room. Opened riot via the desktop riot app, went to the room and clicked leave ... It was possible to leave and also to forget the room.
